packages/pipelines/src/commands/reviewapps/disable.ts

```diff
@@ -1,52 +1,67 @@
-import {Command, flags as HerokuFlags} from '@heroku-cli/command'
+import {Command, flags} from '@heroku-cli/command'
 import * as Heroku from '@heroku-cli/schema'
 import cli from 'cli-ux'
 
-export default class ReviewappsDisable extends Command {
-  static description = 'disable review apps or settings on an existing pipeline'
+export default class ReviewappsDisable2 extends Command {
+  static description = 'disable review apps and/or settings on an existing pipeline'
 
   static examples = [
-    '$ heroku reviewapps:disable -p mypipeline -a myapp --autodeploy'
+    `$ heroku reviewapps:disable -p mypipeline --autodeploy --autodestroy
+`,
   ]
 
   static flags = {
-    app: HerokuFlags.app({description: 'parent app used by review apps', required: true}),
-    pipeline: HerokuFlags.pipeline({required: true}),
-    remote: HerokuFlags.remote(),
-    autodeploy: HerokuFlags.boolean({description: 'disable autodeployments'}),
-    autodestroy: HerokuFlags.boolean({description: 'disable automatically destroying review apps'})
+    pipeline: flags.string({char: 'p', description: 'name of pipeline', required: true}),
+    autodeploy: flags.boolean({description: 'disable autodeployments', required: false}),
+    autodestroy: flags.boolean({description: 'disable automatically destroying review apps', required: false}),
   }
 
   async run() {
-    const {flags} = this.parse(ReviewappsDisable)
+    const {flags} = this.parse(ReviewappsDisable2)
 
-    // if no flags are passed then the user is disabling review apps
-    let disable = !flags.autodeploy && !flags.autodestroy
-
-    let settings = {
-      pull_requests: {
-        enabled: !disable,
-        auto_deploy: false,
-        auto_destroy: false
-      }
+    let settings: {
+      automatic_review_apps: boolean,
+      destroy_stale_apps: boolean,
+      pipeline: string | undefined,
```

Comment:
you can use `pipeline?: string` and same with repo so you don't need to put undefined